signingscript/sign.py

```diff
@@ -485,6 +578,21 @@ def _get_widevine_signing_files(file_list):
     return files
 
 
+# _get_omnija_signing_files {{{1
+def _get_omnija_signing_files(file_list):
+    """Return a dict of path:signing_format for each path to be signed."""
+    files = {}
+    for filename in file_list:
+        fmt = None
+        base_filename = os.path.basename(filename)
+        if base_filename in {'omni.ja'}:
+            fmt = 'omnija'
+        if fmt:
+            log.debug("Found {} to sign {}".format(filename, fmt))
```

Comment:
Do we win anything by structuring like this? (Are we planning on adding other filenames and formats later, or could we have the one `if` and get rid of lines 589 and 590?)

Reply:
I want to allow us to use a different name in the future -- I don't know when it would happen but I don't think we lose much by doing so.